frontend/models/TextClassification.js

```diff
@@ -22,11 +22,20 @@ import _ from "lodash";
 class TextClassificationRecord extends BaseRecord {
   inputs;
 
-  constructor({ inputs, explanation, multi_label, ...superData }) {
+  constructor({
+    inputs,
+    explanation,
+    multi_label,
+    currentAnnotation,
+    ...superData
+  }) {
     super(superData);
     this.inputs = inputs;
     this.explanation = explanation;
     this.multi_label = multi_label;
+    this.currentAnnotation = !currentAnnotation
+      ? this.annotation || null
+      : currentAnnotation;
```

Comment:
it seems that you can make this line in a shorter way: 
this.currentAnnotation = !currentAnnotation || this.annotation || null